src/osd/PrimaryLogPG.cc

```diff
@@ -1888,37 +1888,37 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
 
   // object name too long?
   if (m->get_oid().name.size() > cct->_conf->osd_max_object_name_len) {
-    dout(4) << "do_op name is longer than "
+    dout(4) << __func__ << ": name is longer than "
```

Comment:
Looks : is missed at 1st instance. Does it serve any special purpose. Can you kindly elaborate.
And not including in other instances after function name.